examples/vision/ensemble_of_modified_unets.py

```diff
@@ -0,0 +1,632 @@
mKNHw8oNdT3astdoUDQkNuV1V8UrB9tDKMW9WZv5it62IrUUFBIYU23nFS+Gboe5Ll+R0gMi4KUEo0jeFyjKwIGmTxPcju6RKfptAjLsQpoG7y2QryMINqJoZZCwMtgwqPJkhikJrYlCHrAqjD1tuPALyMz/E3m4PXnXzmYoUBC5J4caB0AxU0WY1erydKQR+Ao7o+SCPktjvzeVPbLi1MDUgTeRS715MkokxrlFadDMYPUBLcienxuW2Kk3V6CFGrUOXS59dcMXoWZ2CTQY8A101JckYUfbJ5XoYJYoAWRRGvRRE6eREg99yZeR0wC1li2OVItqTNq9VcHvzo4ElGU94yeEhxFXpZEbiYSPjqDVEKyjKMwwowiXAov9u9fRorHcWl4c8uTT/5KeYFUgRQ+Mw2DOBJTypeZJnkGqYluSE8Xcpz6tDCXUu/4gibp94tj9GZRcG0Tj438qcRagkDr0lQZqbBj6/979OgJeXc3rjROHUjNFEDR8cYSWJqmaGapGdnveVIAx32d5WDtrCWonlCj7wlc+xRNz4qT/GFZmoM7b94bPRbN3fwgNovkD4Y20E+kFVri8b333BM+Ze/NV6O8ljqQQi4VS1IcEUc/LvTQiHFxbP2U2WOQmuCmIpDVWuelttIipApviyz4XZC+4A4I8R7kQWLv4eGu1SvDy32Vm5Xpdwv7MjKPIahw1onnnkpfiSuDcS0xXU4cSStX708tSMUZxMfC6WwaQSFEhX3YnM1ZU1+HQWqCaU749tk1NXRJnMSlbsOYo5F0B97Mqf9mD3YvXuDhTuWxSKVF6Q/iF/1fXnx/m6DmVDw8lbjW6o9xKUlqcCm60pCrwvBqCucC4BkqUU4XYBA3Uakzwc1xAS7HIDWxRMfoOQ39pK0M3RtRSMX68Vv5rCdgNX/26BMNTrxrSJUYUivTH8x555230Zn5qzswRtVJ5FIJ0iKEVAoN/AgMtRYjZsIaPkKyXZ2WQWTXdCIK/7gQNIgxosRZXPxB9KHXUWlOl3I0wpXXoJtZnOPFPaO3CF8+SROD1EQ3DDZ1Gk5AiuqJOD9+6fdV+R999HfUlbqpHO97V0L+g5B6AWNunMI4BTScDnKpBIhauWj136tThUtpPMaC3KGhYJEvTfTjSSVEXzBC+PAYsIxCSgSKclAYzhP4Tk8FGgNSqZ8LOfwfMUhF6DfdM+J6U/ElR3Cplelbjz52+JHDNODHh4/pDuSPxqV+8sIpmjBWAXo1fyEhpBakEJcKGTp0X/vbW6t73CVDImkwuKQeJ8S9YLNVmwch+3irPU9ASPEClDa5ymFwwBUQ1pX5OgfqnWW4MqoH7fvTwIRxEBugoKmKgHUpg9R0R5Ly/WYVldLdjGNU46cMM5B+39HPHnlEjKHz4WPBA6MP/F5AHkUZ1a5Th15+KSGkkEtRWSoVrCfQQqTC19vQ4Wr3l/ttjjaq6OM8fWajzQ/eilJHnaa+NNfUQwd+aRg7tGS4yllXog1a+QrTsKumsm4QLGXI1jhwFUFBsL1vKR+CPsallD1uOuc5aM/t7h3AOJjUIAknl5SQQi71nbrPfv27N2jsxBLU5o028PvJCxhIkYaGOvT5S1xiWWrBonSq8VtxcfJTkxO8fdW6Ev2Qv6i1wdFynEKKh6JOMhjEFVAAwRIw2swmyqVCNNwhjdULiK16dNeirYOyPj6Iaw7RzA8hBf115i5II6cZpJK/4SfmCXGFhm+wGjyWdWFLdCWkFqB6or1oNqonLMOgxv4z2sDv7mOfYKBdGqAKYB9JzKVWrqbzUnMWT8xzT+pVuDZHRXsw0Frm0JXUOPFTQiFl6+TaEFJ2rlBX2hYs6ehEbxRCCHBZTBq40M5x2D+rTssN13GD5q9NAmVsyKUIqa9w8yQEp5cKTD0xqY2WNBcnsKHV4eCKmzyomML1UjFcagFG7kBtlb8HFwOhLgvnpRKqJ1CJjvOi77/9zjZct5vBI6QUaviIpkLkUqmxXkoFtvLidl+vr6J4+T6KDhFSJtKmV39UBoXBNlxaaHdBgwXRVK3r9nja/ZqQycJXaIXhiq5CV50PIYiQMgeAb7XWVCOXqq7B/SRNTOM3sQ1DcHWHv9c8SD+idFWvkkulY+SOEF+oUWFQKrFb3Zl+cwQiigyd6uW2IaDEqd5RuNTNKEvBVEeJGgetVM52HLihUbB4DQzcZREGzVB2GkKuQW3QUmMRcAUmj/zd3Orjqt2tSws8QSc4dQVgxVVTIpfyLgGfC/qDG1A9USar48fxSJN0KoPUBBMWV/Cc0Bag+R0mZFPHPk1fEEkYgaAZj4QtbyiXWhk5Fs2sXH03/SqLAaqQT6EsFT0Wza1EWSpDdBczwS8wGZdDD7sZ8G0jqCQeRW9RmQ2ebshDSnV7IDtXyIa8PPHWuRs86DOgCtmSJg2rECjMDj9Sdhd0o+lJdwGeLdUNH0iuDYPURLeH+BmWvsUZORy542c/jqZtqAOMjFgwKJvyWLTWz1CEipjNCrD7J9FDitzPtkKqeEgSSYwylChGJaC3RC3xgMR8sEC5BlpQniMOHJUFyZZnkJrwFhGi9qCLM+nYbXxJ2beUVxIgc3OcJarycPLlSQwp8LVwcExfDpeHoWQpmUXgU4vLxURGTpFGwSUjLlyHagzpecmaGKQmtWUW58xZLC0YFH/jIIDgS5hiq6kT1snYnJMaK3onlb7JeHEGqcltFQ6DAYj+zKlP84lLyzZnzsEI9ywlIQUYpCa7UVTLNh+b8LT5YlRgsJSUFGCQSspmYQ+VuhRgkErdtmNPnpQUYJBKymZhD5W6FGCQSt22Y0+elBRgkErKZmEPlboUYJBK3bZjT56UFGCQSspmYQ+VuhRgkErdtmNPnpQUYJBKymZhD5W6FGCQSt22Y0+elBRgkErKZmEPlboUYJBK3bZjT56UFGCQSspmYQ+VuhRgkErdtmNPnpQUYJBKymZhD5W6FEBI/f+1KifGjvFGrQAAAABJRU5ErkJggg==)
```

Comment:
Please upload images to imgur and add the url to the file, rather than using base64 encoding. In the current state I cannot review the PR.

Reply:
Thank you for responding, I will add the URL.